projects/plugins/backup/readme.txt

```diff
@@ -104,7 +104,7 @@ Jetpack’s off-site backup storage allows you to restore a clean version of you
 
 = How do I create a WordPress backup for my site? =
 
-If you don’t have Backups as part of your Jetpack plan, go to your WP Admin and then go to **Jetpack > Dashboard > Plans**. Depending on your plan, you can get daily or real-time WordPress backups. Daily backups are archived for 30 days, and real-time backups have unlimited storage.
+If you don’t have Backups as part of your Jetpack plan, go to your WP Admin and then go to **Jetpack > Dashboard > Plans**.
```

Comment:
Since this is the description for the Backup plugin, should we take that opportunity to change that sentence so it doesn't refer to a path that's specific to the Jetpack plugin and not available in the Backup plugin?

Reply:
@jeherve Great catch!

Reply:
EIther that, or maybe recommend that they follow the prompt in the Backup menu to upgrade?

Reply:
@simonktnga8c, calling you in for a copy recommendation since we have this same change on Jetpack.com 🙂  What could we put here to give a good experience to people who download the Backup plugin?

Reply:
Just wanted to mention if we give plugin specific guidelines, the screen we've screen shotted in this PR will be outdated by the next Jetpack Backup version.  You can see https://github.com/Automattic/jetpack/pull/21646 for reference to the new screen.  Mainly we'll have a `Get Jetpack Backup` button instead of `Upgrade Now`

Reply:
Yeah I would link out to Jetpack.com as @keoshi suggested. 

Copy suggestion:

"If you don’t have Backups as part of your Jetpack plan, visit [Jetpack.com to learn more and purchase](https://jetpack.com/upgrade/backup/).'